packages/core-contracts/contracts/token/ERC721Storage.sol

```diff
@@ -6,10 +6,14 @@ contract ERC721Storage {
         string name;
         string symbol;
         string baseTokenURI;
+        uint256[] allTokens;
```

Comment:
Add all enumerable storage related (all the additions) to a specific storage i.e. ERC721EnumerableStorage.sol so that it doesn't add complexity to non-enumerable ERC721's

Reply:
Also, this change won't pass the deployer since you are adding variables in the middle of the storage struct (if it passed, let us know, it shouldn't).
Any new variable should be added at the bottom of the storage struct. 

On any upgrade this contract contract will have the state data corrupted. Let's jump in a call if you need more info about that.